dali/test/python/test_operator_warp.py

```diff
@@ -163,35 +163,39 @@ def test_cpu_vs_cv():
   for batch_size in [1, 4, 19]:
     for use_input in [False, True]:
       for (itype, otype) in io_types:
-        print("Testing cpu vs cv",
-              "\nbatch size: ", batch_size,
-              " matrix as input: ", use_input,
-              " input_type: ", itype,
-              " output_type: ", otype)
-        cv_pipeline = CVPipeline(batch_size, otype, itype, use_input);
-        cv_pipeline.build();
+        for inverted in [False, True]:
+          print("Testing cpu vs cv",
+                "\nbatch size: ", batch_size,
+                " matrix as input: ", use_input,
+                " input_type: ", itype,
+                " output_type: ", otype,
+                " map_inverse:", inverted)
+          cv_pipeline = CVPipeline(batch_size, otype, itype, use_input, inverted=inverted);
+          cv_pipeline.build();
 
-        cpu_pipeline = WarpPipeline("cpu", batch_size, otype, itype, use_input);
-        cpu_pipeline.build();
+          cpu_pipeline = WarpPipeline("cpu", batch_size, otype, itype, use_input, inverted=inverted);
+          cpu_pipeline.build();
 
-        compare(cv_pipeline, cpu_pipeline, 8)
+          compare(cv_pipeline, cpu_pipeline, 8)
 
 def test_gpu_vs_cv():
   for batch_size in [1, 4, 19]:
     for use_input in [False, True]:
       for (itype, otype) in io_types:
-        print("Testing gpu vs cv",
-              "\nbatch size: ", batch_size,
-              " matrix as input: ", use_input,
-              " input_type: ", itype,
-              " output_type: ", otype)
-        cv_pipeline = CVPipeline(batch_size, otype, itype, use_input);
-        cv_pipeline.build();
-
-        gpu_pipeline = WarpPipeline("gpu", batch_size, otype, itype, use_input);
-        gpu_pipeline.build();
-
-        compare(cv_pipeline, gpu_pipeline, 8)
+        for inverted in [False, True]:
+          print("Testing gpu vs cv",
+                "\nbatch size: ", batch_size,
+                " matrix as input: ", use_input,
+                " input_type: ", itype,
+                " output_type: ", otype,
+                " map_inverse:", inverted)
+          cv_pipeline = CVPipeline(batch_size, otype, itype, use_input, inverted=inverted);
```

Comment:
Nitpick: Use `test_utils.compare_pipelines`?

Reply:
it's used inside compare functions